scripts/release-notes.py

```diff
@@ -43,6 +43,22 @@ def num_commits_since_prev_tag(token, base_url):
     print(f"There are {num_commits} new commits since {prev_release_tag}")
     return num_commits
 
+categories = [
+    {'title': '#### ⛔ Breaking Changes', 'label': 'changelog:breaking-change'},
+    {'title': '#### New Features', 'label': 'changelog:new-feature'},
+    {'title': '#### Bug fixes, Minor Improvements', 'label': 'changelog:bugfix-or-minor-feature'},
+    {'title': '#### 🚧 Experimental Features', 'label': 'changelog:exprimental'},
+    {'title': '#### CI Improvements', 'label': 'changelog:ci'},
+    {'title': '', 'label': 'changelog:test'},
+    {'title': '', 'label': 'changelog:skip'}
```

Comment:
`'title': None` would be more idiomatic Python.

Reply:
Makes sense. I am writing Python code for the first time :)